bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/cannon/Cannon.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * File: cannon.java
+ * Class: Cannon
+ *
+ * Version: 0.0.1
+ * Date: September 8th, 2016
+ */
+
+package nl.tudelft.bust_a_move20162017.bust_a_move_framework.cannon;
+
+import nl.tudelft.bust_a_move20162017.bust_a_move_framework.bubble.Bubble;
+
+import java.awt.Color;
+import java.awt.Graphics;
+
+/**
+ * The Cannon class represents a cannon entity.
+ *
+ * @author Maurice Willemsen
+ */
+
+public class Cannon {
+
+	private int ANGLE;
+
+	private Bubble.ColorChoice currBubbleColour;
```

Comment:
Store a bubble instead of the bubble color. If you store the bubble, you can also access its color.